### PR TITLE
Some fixes for FX pass with nvFuser backend

### PR DESCRIPTION
**Pull request by SherlockNoMad**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #81911

### CODE REVIEW

File: torch/fx/passes/backends/nvfuser.py [hunk 1]
```diff
@@ -266,15 +266,16 @@ def compile(self, graph_module: GraphModule) -> GraphModule:
             logging.debug("partitioner_cache hit!")
             fused_graph_module = self.partitioner_cache[graph_module]
         else:
-            partitioner = CapabilityBasedPartitioner(graph_module, self.supported_ops)
+            partitioner = CapabilityBasedPartitioner(
```
Comment (jjsjann123) [inline torch/fx/passes/backends/nvfuser.py:269]: If we are cleaning/tinkering the fx execution, maybe we also want to cherry-pick this guy https://github.com/pytorch/pytorch/pull/81820/files#diff-8423e87cb2c44e61ff9701c15b9f55a6f9614066dcf1e0ce8c35d571db6d0f15R207-R208

File: torch/fx/passes/backends/nvfuser.py [hunk 2]
```diff
@@ -266,15 +266,16 @@ def compile(self, graph_module: GraphModule) -> GraphModule:
...
+                graph_module, self.supported_ops, allows_single_node_partition=False)
             fused_graph_module = partitioner.partition_and_fuse()
 
             self.partitioner_cache[graph_module] = fused_graph_module
 
         # Overriding fused_module's __call__() function with lower_to_prims_and_execute()
         for node in fused_graph_module.graph.nodes:
             # TODO: use a better way to identify fused submodule
-            if "fused_" in node.name:
+            if node.op == "call_module" and "fused_" in node.name:
```
Comment (jjsjann123) [inline torch/fx/passes/backends/nvfuser.py:278]: Just curious: any place where we are actually getting into a `fused_` where it's not a `call_module`?

**Review by jjsjann123 — APPROVED** [review]
LGTM with a few questions.

File: torch/fx/passes/backends/nvfuser.py (hunk 2 above)
Comment (SherlockNoMad) [reply torch/fx/passes/backends/nvfuser.py:278]: yes, running into a node call "aten._fused_moving_avg_obs_fq_helper_functional.default"

**Review by IvanYashchuk — APPROVED** [review]
Looks good!

Soon after https://github.com/pytorch/pytorch/pull/81764 is merged we'll be able to move away from the static aten2prim table here.

**Review by davidberard98 — APPROVED** [review]

**Pull request closed without merging**